api.bs

```diff
@@ -608,12 +608,12 @@ The arguments to <a method for=PrivateAttribution>saveImpression()</a> are as fo
     there is a way to match any filter data, which would not be possible if
     there were a default integer value.
   </dd>
-  <dt><dfn>conversionSite</dfn></dt>
+  <dt><dfn>conversionSites</dfn></dt>
   <dd>
-    The site where [=conversions=] for this impression may occur, identified by
-    its domain name. The <a method for=PrivateAttribution>measureConversion()</a>
-    method will only attribute to this impression when called by the indicated
-    site.
+    The sites where [=conversions=] for this impression may occur, identified by
+    their domain names. The <a method for=PrivateAttribution>measureConversion()</a>
+    method will only attribute to this impression when called by one of the indicated
+    sites. If empty, any site will match.
```

Comment:
Do we really want this to be empty?  Is there any situation where it makes sense to avoid specifying this?

Reply:
If you'd prefer, we could instead make the empty list an error.

CC @csharrison

Reply:
How about we take this (which is unambiguously OK), then we can put the question about empty/not to the broader group.